### ISSUE TITLE
elm-format 0.8.5 breaks elm-upgrade

### ISSUE THREAD

**Issue by reinefjord** [avh4/elm-upgrade#93]
[elm-format 0.8.5](https://github.com/avh4/elm-format/releases/tag/0.8.5) has removed the `--upgrade` option, breaking elm-upgrade:

```
INFO: Upgrading *.elm files in ./
Invalid option `--upgrade'

elm-format 0.8.5

Usage: elm-format [INPUT] [--output FILE] [--yes] [--validate] [--stdin] 
                  [--elm-version VERSION]
```

**Comment by avh4** [comment]
Do a lot of folks still have 0.18 projects they're upgrading?   There's one more feature I have to finish before I can make the elm-refactor binaries.

The workaround is to use elm-format-0.8.4 temporarily when running elm-upgrade.

**Comment by reinefjord** [comment]
I have one rather large project that's still on 0.18. Installing 0.8.4 is fine for me, just wanted to make you aware :)

**Comment by mbylstra** [comment]
> Do a lot of folks still have 0.18 projects they're upgrading? There's one more feature I have to finish before I can make the elm-refactor binaries.
> 
> The workaround is to use elm-format-0.8.4 temporarily when running elm-upgrade.

Yes, probably for years to come I'm afraid. I have loads of old personal projects that eventually I come back to (nothing my salary depends on). Still, it wasn't too hard to find this issue and do the workaround.
